website_and_docs/content/documentation/webdriver/getting_started/first_script.en.md

```diff
@@ -0,0 +1,203 @@
+
+        // 5. Send command to take actions on elements
+        searchBox.SendKeys("Selenium");
+        searchButton.Click();
+
+        // 6. Request information about an element
+        driver.FindElement(By.Name("q")).GetAttribute("value"); // => "Selenium"
+
+        // 7. End the session
+        driver.Quit();
+    }
+}
+{{< /tab >}}
+{{< tab header="Ruby" >}}
+require 'selenium-webdriver'
+
+    # 1. Start the session
+driver = Selenium::WebDriver.for :chrome
+
+    # 2. Send command for browser to navigate
+driver.get 'https://google.com'
+
+    # 3. Request information about the browser
+driver.title # => 'Google'
+
+    # 4. Send command to find elements
+search_box = driver.find_element(name: 'q')
+search_button = driver.find_element(name: 'btnK')
+
+    # 5. Send command to take actions on elements
+search_box.send_keys 'Selenium'
+search_button.click
+
+    # 6. Request information about an element
+driver.find_element(name: 'q').attribute('value') # => "Selenium"
+
+    # 7. End the session
+driver.quit
+{{< /tab >}}
+{{< tab header="JavaScript" >}}
+const {Builder, By, Key, until} = require('selenium-webdriver');
+
+(async function helloSelenium() {
+    // 1. Start the session
+    let driver = await new Builder().forBrowser('chrome').build();
+
+    // 2. Send command for browser to navigate
+    await driver.get('https://www.google.com');
+
+    // 3. Request information about the browser
+    await driver.getTitle(); // => "Google"
+
+    // 4. Send command to find elements
+    let searchBox = await driver.findElement(By.name('q'));
+    let searchButton = await driver.findElement(By.name('btnK'));
+
+    // 5. Send command to take actions on elements
+    await searchBox.sendKeys('Selenium');
+    await searchButton.click();
+
+    // 6. Request information about an element
+    await driver.findElement(By.name('q')).getAttribute("value"); // => 'Selenium'
+
+    // 7. End the session
+    await driver.quit();
+})();
+{{< /tab >}}
+{{< tab header="Kotlin" >}}
+import org.openqa.selenium.By
+import org.openqa.selenium.chrome.ChromeDriver
+
+fun main() {
+    // 1. Start the session
+    val driver = ChromeDriver()
+
+    // 2. Send command for browser to navigate
+    driver.get("https://google.com")
+
+    // 3. Request information about the browser
+    driver.getTitle(); // => "Google"
+
+    // 4. Send command to find elements
+    val searchBox = driver.findElement(By.name("q"));
+    val searchButton = driver.findElement(By.name("btnK"))
+
+    // 5. Send command to take actions on elements
+    searchBox.sendKeys("Selenium");
+    searchButton.click();
+
+    // 6. Request information about an element
+    driver.findElement(By.name("q")).getAttribute("value"); // => "Selenium"
+
+    // 7. End the session
+    driver.quit()
+}
+{{< /tab >}}
+{{< /tabpane >}}
+
+{{% pageinfo color="warning" %}}
+Coming soon:
+* Link to advanced code with a complex form filling example
+  {{% /pageinfo %}}
```

Comment:
I would remove this, I see "coming soon" comments as something we might never have :)